### ISSUE TITLE
How Do I Post Image,Text And Link Using Tweetsharp

### ISSUE THREAD

**Issue by FaiqueAli** [danielcrenna/tweetsharp#182]
I am posting image with text on twitter using tweetshare api and getting ok response, but unable to make these info with hyperlink so that user can see full info at mywebsite 
here is my code.

Bitmap img = new Bitmap(Server.MapPath("~/img/test.jpg"));
 
MemoryStream ms = new MemoryStream();
               img.Save(ms, System.Drawing.Imaging.ImageFormat.Jpeg);
               ms.Seek(0, SeekOrigin.Begin);
               Dictionary<string, Stream> images = new Dictionary<string, Stream> { { "mypicture", ms } };
 
               var result = service.SendTweetWithMedia(new SendTweetWithMediaOptions { Status = "this is my second tweet", Images = images });

can any one help me please..

**Comment by FaiqueAli** [comment]
wonder to see as there is no any response even after 19 hours..
Is this forum is alive??

**Comment by Yortw** [comment]
No. TweetSharp is no longer maintained by the author, as noted in the readme.md file in the repo root. Occasionally someone does respond here, but it given the repo is effectively abandoned the responses tend to be slow, if they come at all.

With regards to your problem; It is not clear exactly what you want to do. It sounds like you want to make the text and image in your tweet hyperlinks to an external site. Twitter does not support this (at least not as far as I know - I have never seen it done, and cannot see a way to do it either with the Twitter API or Twitter website). 

What you *can* do is include a link to your website in the tweet, and have Twitter turn the url text into a link for you. To do that, just add the url into the text, i.e 

Status = "this is my second tweet, see http://www.mywebsite.com for more"

Twitter will automatically detect the url and make it a hyperlink anywhere it displays a Tweet. It will also provide details of the link and it's location within the Tweet to any Twitter clients that retrieve the tweet, but it is up to the client to decide if it wants to make it a hyperlink or leave it as text.

You should see the Twitter API documentation (dev.twitter.com) for information about link handling/wrapping and how to determine how many characters in a Tweet link will take up (there is an endpoint that returns the current length for http and https urls).
